templates/scripts/add-vpn-user.sh

```diff
@@ -16,7 +22,7 @@ if [ -z "$POD" ]; then
 fi
 
 function k8s_exec() {
-  kubectl -n vpn exec -it $POD -- /bin/bash -c "$1"
+  kubectl -n vpn exec $POD wireguard --container wireguard -- /bin/bash -c "$1"
```

Comment:
Does the wireguard pod have multiple containers in it?

Reply:
yeah somehow it has a `sysctls` container in there